src/pudl/analysis/plant_parts_eia.py

```diff
@@ -713,12 +685,8 @@ def add_one_to_many(
             ppl=plant_parts_eia,
             part_name="plant_gen",
             cols_to_keep=["plant_part"],
-        )[["record_id_eia_og", "record_id_eia", "plant_part_og"]].rename(
-            columns={
-                "record_id_eia": "gen_id",
-                "record_id_eia_og": "record_id_eia",
-                "plant_part_og": "plant_part",
-            }
+        )[["record_id_eia", "record_id_eia_plant_gen", "plant_part"]].rename(
+            columns={"record_id_eia_plant_gen": "gen_id"}
```

Comment:
i changed the `match_to_single_plant_part` function just a little to add a suffix to the plant_gen side of the merge instead of adding the suffix of `_og` to the `multi_gran_df` side

Reply:
its just a merge suffix change within `match_to_single_plant_part`. I've made sure all of the use cases of `match_to_single_plant_part` all have the same names coming out of it, which was simple because its only being used two other times and they both already had renames here. The renames are now more simple bc as you can see we were removing the `_og` suffix before.